agreement/proposal.go

```diff
@@ -47,7 +48,7 @@ type transmittedPayload struct {
 	_struct struct{} `codec:",omitempty,omitemptyarray"`
 
 	unauthenticatedProposal
-	PriorVote unauthenticatedVote `codec:"pv"`
+	PriorVote msgp.Raw `codec:"pv"`
```

Comment:
```
panic: msgpack decode error [pos 46]: cannot decode unsigned integer: unrecognized descriptor byte: a4/string|bytes

goroutine 255 [running]:
github.com/algorand/go-algorand/agreement.(*testingNetwork).multicast(0xc0001b01c0, {0x1377d11, 0x2}, {0xc003436700, 0x319, 0x688}, 0x0, 0x0)
	/opt/cibuild/project/agreement/service_test.go:210 +0x1585
```

```
		if tag == protocol.ProposalPayloadTag {
			r := bytes.NewBuffer(data)

			var tp transmittedPayload
			err := protocol.DecodeStream(r, &tp)
			if err != nil {
				panic(err)
			}
```

Looks like it tries to decode `pv` field and gets only an encoded value and not encoded key-value pair?